10.js

```diff
@@ -0,0 +1,23 @@
+const resolveSymlinks = input => {
+  const inputArray = input.split('\n')
+  let symlinks = []
+  let path = inputArray[inputArray.length - 1]
+
+  for (let index = 1; index <= inputArray[0]; index++) {
+    symlinks.push(inputArray[index])
+  }
+
+  symlinks = symlinks.map(symlink => symlink.split(':'))
+
+  symlinks.forEach(symlink => {
+    const symlinkRegExp = new RegExp(symlink[0])
+    let actualPath = symlink[1]
+    if (actualPath.slice(-1) === '/') actualPath = actualPath.slice(0, -1)
```

Comment:
I recommend using explicit blocks for clarity.  Having said that, these two lines can be simplified, and use string built ins for clarity:
```js
const actualPath = symlink[ 1 ].endsWith( '/' ) ? symlink[ 1 ].slice( 0, -1 ) : symlink[ 1 ]
path = path.replace( regex, actualPath )
```
I suspect there's an opportunity to turn this into a reduce as well...